.github/workflows/AirspeedVelocity.yml

```
@@ -0,0 +1,78 @@
+name: Benchmark a Pull Request
+
+on:
+  pull_request_target:
+    branches:
+      - master
+
+permissions:
+  pull-requests: write
+
+jobs:
+    generate_plots:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - uses: julia-actions/setup-julia@v1
+              with:
+                version: "1.8"
```

```suggestion
                version: "1.10"
```
We require at least julia v1.9. So I would go with either v1.9 or v1.10 as we use in the CI tests.

---

Ok, now the Action fails because of this.